src/Parser/DemonHunter/Vengeance/Modules/Spells/SigilOfFlame.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import Analyzer from 'Parser/Core/Analyzer';
+import AbilityTracker from 'Parser/Core/Modules/AbilityTracker';
+import Enemies from 'Parser/Core/Modules/Enemies';
+
+import SPELLS from 'common/SPELLS/index';
+import SpellIcon from 'common/SpellIcon';
+
+import { formatPercentage, formatThousands, formatDuration } from 'common/format';
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class SigilOfFlame extends Analyzer {
+  static dependencies = {
+    abilityTracker: AbilityTracker,
+    enemies: Enemies,
+  };
+
+  previousBuff = 0;
+  currentBuff = 0;
+  difference = 0;
+  stacked = 0;
+
+  on_byPlayer_applydebuff(event){
+    const spellId = event.ability.guid;
+
+    if(spellId !== SPELLS.SIGIL_OF_FLAME_DEBUFF.id) {
+      return;
+    } else{
+      if(this.previousBuff === 0){
+        this.previousBuff = event.timestamp;
+        return;
+      }else{
+        this.currentBuff = event.timestamp;
+        this.difference = this.currentBuff - this.previousBuff;
```

Comment:
This value is not used outside this method or re-used at a later time (it's only overwritten), so please remove it as a class property and instead define a local variable.